### ISSUE TITLE
Cannot pull grafana/grafana:11.1.3 from dockerhub

### ISSUE THREAD

**Issue by flori** [grafana/grafana#91190]
### What happened?

```
$ docker pull grafana/grafana:11.1.3
Error response from daemon: manifest for grafana/grafana:11.1.3 not found: manifest unknown: manifest unknown
```

### What did you expect to happen?

```
$ docker pull grafana/grafana:11.1.3
11.1.3: Pulling from grafana/grafana
…
```

### Did this work before?

Not to my knowledge.

### How do we reproduce it?

1. docker pull grafana/grafana:11.1.3

### Environment (with versions)?

Grafana:
OS:
Browser:

### Grafana platform?

Docker

**Comment by guusvw** [comment]
Same for Grafana `10.4.6`, looks like both versions are only available under `grafana-oss` & `grafana-enterprise`

https://hub.docker.com/layers/grafana/grafana-oss/11.1.3/images/sha256-7fe6d7cfa092e62ca3e893c7cf595417e78534ade2b65a49f2754fb0ecb6b792?context=explore

**Comment by flori** [comment]
Somebody seems to have fixed it, I can pull `11.1.3` and `10.4.6` now.
